pypet2bids/pypet2bids/ecat2nii.py

```diff
@@ -132,7 +132,11 @@ def ecat2nii(ecat_main_header=None,
             prompts.append(0)
             randoms.append(0)
 
-    final_image = img_temp * main_header['ECAT_CALIBRATION_FACTOR']
+    ecat_cal_units = main_header['CALIBRATION_UNITS']  # Header field designating whether data has already been calibrated
+    if ecat_cal_units==0:                              # Calibrate if it hasn't been already
+        final_image = img_temp * main_header['ECAT_CALIBRATION_FACTOR']
+    elif ecat_cal_units==1:                            # And don't calibrate if it has
```

Comment:
```suggestion
    if ecat_cal_units==1:                              # Calibrate if it hasn't been already
        final_image = img_temp * main_header['ECAT_CALIBRATION_FACTOR']
    else:                            # And don't calibrate if CALIBRATION_UNITS is anything else but 1
```

Reply:
I'm going to apply this locally quickly and then commit it if the results match up with the changes Cyril just pushed to the matlab branch.

Reply:
It's good! 
<img width="747" alt="Screenshot 2024-02-16 at 11 51 54 AM" src="https://github.com/openneuropet/PET2BIDS/assets/28850131/6d7678e5-5eda-40a2-b4e7-b0b94d10e66b">